pom.xml

```diff
@@ -223,7 +223,7 @@
     <aws-secretsmanager-caching.version>1.0.1</aws-secretsmanager-caching.version>
     <aws-java-sdk.version>1.12.499</aws-java-sdk.version>
     <jansi.version>2.4.0</jansi.version>
-    <spring.version>5.2.24.RELEASE</spring.version>
+    <spring.version>5.3.21</spring.version>
```

Comment:
Should we add a comment here to remind that if someone upgrade `spring `dependency, they also should check its sub-dependency atlas  & upgrading `atlas `version in `ql/pom.xml`?